acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -46,4 +46,8 @@
 }
 
 install_packages_on(master, MASTER_PACKAGES)
-install_packages_on(agents, AGENT_PACKAGES)
+if ENV['AIO_AGENT_INSTALL'].nil?
```

Comment:
Along with the repo comment i left below, it seems like this could be...

```
if ENV['AIO_AGENT_INSTALL']
  install_packages_on(agents, 'puppet-agent')
else
  install_packages_on(agents, AGENT_PACKAGES)
end
```

That assumes the correct repo has already been added for the puppet-agent.